documentation-site/pages/components/dnd-list.mdx

```diff
@@ -79,35 +82,11 @@ This component also supports iOS and Android browsers and screen readers.
   <OverrideLabel />
 </Example>
 
-## Overrides
-
-<Overrides
-  name="Drag and Drop List"
-  component={ListExports}
-  renderExample={props => (
-    <StatefulList
-      removable
-      initialState={{items: ['Item 1', 'Item 2', 'Item 3', 'Item 4']}}
-      overrides={props.overrides}
-    />
-  )}
-/>
-
 ## API
 
-<API
-  heading="List API"
-  api={require('!!extract-react-types-loader!../../../src/dnd-list/list.js')}
-/>
-
 <API
   heading="Stateful List API"
   api={require('!!extract-react-types-loader!../../../src/dnd-list/stateful-list.js')}
 />
 
```

Comment:
I would delete this too (+ the import). The preferred way is to use stateless API and there is still a stateful example.

(we might be able to delete `extract-react-types-loader` too)

Reply:
Since we don't describe the rest of available components API, like stateful when only use stateless in the yeard, and subcomponents like in the Tabs and Accordion cases, I think we should keep the API docs for those not included in yard.